python/cuml/test/test_random_forest.py

```diff
@@ -340,13 +340,24 @@ def test_rf_regression_float64(large_reg, datatype):
                                        convert_dtype=False)
 
 
+def check_predict_proba(test_proba, baseline_proba, X_test, y_test, rel_err):
+    y_proba = np.zeros(np.shape(baseline_proba))
```

Comment:
The shape of  the predicted probability variable for multi class classification will be (nrows, n_classes). The way y_proba is defined below it will only work as intended for binary classification problems and not multi class classification.